website/source/docs/guides/deployment.html.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: "docs"
+page_title: "Production Deployment"
+sidebar_current: "docs-guides-deployment"
+description: |-
+  Best practice approaches for Consul production deployments.
+---
+
+# Consul Production Deployment Guide
+
+As applications are migrated to dynamically provisioned infrastructure, scaling services and managing the communications between them becomes challenging. Consul’s service discovery capabilities provide the connectivity between dynamic applications. Consul also monitors the health of each node and its applications to ensure that only healthy service instances are discovered. Consul’s distributed runtime configuration store allows updates across global infrastructure.
```

Comment:
We may want to drop this as it could change slightly down the road, and is more an "about consul". Maybe a link to the intro or other existing section?

Reply:
Addressed in b198a5fa3